bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/dialogs/ViewsPreferencePage.java

```diff
@@ -192,6 +200,32 @@ protected Control createContents(Composite parent) {
 		return comp;
 	}
 
+	private void createUpdateOnRuntimeButton(Composite composite) {
+		if (!OS.isWindows()) {
+			return;
+		}
+		createLabel(composite, ""); //$NON-NLS-1$
+		Group buttonComposite = new Group(composite, SWT.LEFT);
+		buttonComposite.setText("HiDPI settings"); //$NON-NLS-1$
+
+		GridData gridData = new GridData(SWT.FILL, SWT.CENTER, true, false);
+		gridData.horizontalSpan = ((GridLayout) composite.getLayout()).numColumns;
+		buttonComposite.setLayoutData(gridData);
+		buttonComposite.setFont(composite.getFont());
+		GridLayout layout = new GridLayout(1, false);
+		buttonComposite.setLayout(layout);
+		Label infoLabel = new Label(buttonComposite, SWT.WRAP);
+		infoLabel.setText(
+				"This preference is Windows only. Activating it will change the DPI awareness of the IDE to PerMonitorV2 and dynamically scale all windows according to the monitor it is currently assigned to. An additional effect of this preference is to set the default browser to Edge to provide the appropriate scaling of content displayed in a browser. This feature is still in development and therefore marked as experimental."); //$NON-NLS-1$
```

Comment:
Shouldn't we move this text to a messages.properties file?

I would write:
"This setting is specific to the Windows operating system..."

Reply:
The only reason I didn't bother to put the message in the file is that later it would be easier to remove this temporary option (just from one place) but yeah  I will move it.